onnxruntime/core/graph/graph.cc

```diff
@@ -3815,6 +3823,77 @@ bool Graph::GetOrtValueInitializer(const std::string& name, OrtValue& value, boo
   return false;
 }
 
+Status Graph::LoadExternalInitializerAsOrtValue(const std::string& name, OrtValue& value) {
+  auto tensor_proto_iter = name_to_initial_tensor_.find(name);
+  ORT_RETURN_IF(tensor_proto_iter == name_to_initial_tensor_.end(), "Cannot load unknown initializer named '",
+                name, "'.");
+  const ONNX_NAMESPACE::TensorProto& tensor_proto = *tensor_proto_iter->second;
+
+  ORT_RETURN_IF(!utils::HasExternalDataInFile(tensor_proto), "Initializer '", name, "' does not have external data.");
```

Comment:
If someone calls this and there's in-memory external data should that be an error? 

Should we do this after checking if there's already a value in `ortvalue_initializers_`?